Propose tweet about Ranger

mfix22 opened the pull request:
This PR adds a new tweet about [Ranger](https://reporanger.com). Let me know if you have any feedback!

mfix22 commented:
@JasonEtco doh! I didn't even realize I did that. I will make both updates (:

JasonEtco commented:
Thanks @mfix22 ❤️ and unrelated to anything else - I know that @dawnlabs is responsible for Carbon, I'm a big fan ✨

mfix22 commented:
Hey @JasoEtco, thanks so much for the kind words 🙏We love that a lot of the GitHub folks use Carbon ❤️ 

Just updated our listing description to include the markdown you pasted above, and I have updated the tweet as well 🙂. Looking into how we can include it on the site as well, but that might take little longer. 

Let me know if you have other feedback 👌

gr2m commented on tweets/2019/03/ranger-0.tweet:
```diff
@@ -0,0 +1,3 @@
+Ranger, a Probot app for open-source maintainers, just landed in the @github marketplace! Ranger helps automate common maintenance tasks, like merging PRs, closing issues, creating tags, deleting comments and branches, and more!
+
+Visit reporanger.com to find out more.
```
just to be 100% sure that Twitter will turn this into a link when we post the tweet via its API, maybe add the https?

```suggestion
https://reporanger.com/
```

mfix22 replied:
I can do that! I did paste the text into twitter and it turned it into a link. Is that enough to confirm? 

Otherwise I can just make this update too (:

gr2m replied:
Yeah better safe than sorry 🤷‍♂️

gr2m commented:
ping @JasonEtco

mfix22 commented:
Thank you all :pray:!
